src/Type/InterfaceType/ContentTemplate.php

```diff
@@ -53,6 +53,10 @@ public static function register_content_template_types() {
 
 			if ( ! empty( $replaced_name ) ) {
 				$name = $replaced_name;
+			} else {
+				// If replaced_name is empty, use the file name
+				$file_name = explode( '.', $file )[0] ?? '';
```

Comment:
This is getting flagged (and rightly so) by PHPStan, because we no longer have access to `$file` once the previous loop ends.